src/xperm_rules.md

```diff
@@ -74,6 +75,26 @@ Conditional Policy Statements
 | ----------------------- | ----------------------- | ----------------------- |
 | No                      | No                      | No                      |
 
+### Extended Permission Evaluation
+
+Extended permission rules are evaluated as follows:
+
+* If no extended permissions are defined, only the resource-level policy is
```

Comment:
I don't think "resource-level policy" is clear or exactly precise in this case.  Maybe something like "The standard SELinux checks around AVC rules and constraints will be performed".

Reply:
Thanks for the clarification!; that makes sense, and I have updated the documentation with this wording